x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/collection_email_powershell_exchange_mailbox.json

```diff
@@ -46,5 +47,5 @@
     }
   ],
   "type": "eql",
-  "version": 1
+  "version": 2
```

Comment:
We'll want to increment the version +1 from the rule updates being pushed to 7.11.2 from this PR: https://github.com/elastic/kibana/pull/91553

Otherwise folks upgrading from 7.11.2 -> 7.12 will not get these index changes since the rule is the same version.



![image](https://user-images.githubusercontent.com/2946766/108123374-fd143d00-7062-11eb-9a64-cc2ff7792c5d.png)


https://github.com/elastic/kibana/pull/91553/files#diff-cd628be41e292c3335d54a71ae53950610dedaa6f21ae14443e64af7af963e91R48-R50

Reply:
Note: Version will be re-generated once the `7.11.2` rules are merged (https://github.com/elastic/kibana/pull/91553).